mutalyzer/website/views.py

```diff
@@ -68,6 +68,25 @@ def request_terms():
     return terms
 
 
+@website.app_template_filter('unicode')
+def short_seq_filter(s):
+    return unicode(s)
+
+
+@website.app_template_filter('short')
+def short_seq_filter(s, max_len=21):
+    if len(s) > max_len:
+        return s[:max_len / 2 - 1] + '...' + s[-max_len / 2 + 2:]
+    return s
+
+
+@website.app_template_filter('too_long')
+def short_seq_filter(s, max_len=21):
```

Comment:
Function name should be changed.

I'm also not sure if this should be generalized as a filter. It's only one conditional and the substituted text makes it quite specific to that one use case.